src/components/InfoBox/InfoBox.tsx

```diff
@@ -2,12 +2,12 @@ import classnames from 'classnames';
 import React, { ReactNode } from 'react';
 import Icon from '../Icon/Icon';
 
-interface InfoBoxProps extends Omit<React.HTMLProps<HTMLDivElement>, 'className'> {
+interface InfoBoxProps extends Omit<React.HTMLProps<HTMLDivElement>, 'className' | 'title'> {
```

Comment:
What does this do?

Reply:
So we basically get the error that 
```
Types of property 'title' are incompatible.
    Type 'string | Element | undefined' is not assignable to type 'string | undefined'.
      Type 'Element' is not assignable to type 'string'.
```
because `title` is a property in the extended interface of `Omit<HTMLProps<HTMLDivElement>`. So adding this allows title's types to be overridden below in this block.